makefiles/wabt.mk

```
@@ -0,0 +1,43 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS  += wabt
+WABT_VERSION := 1.0.23
+DEB_WABT_V   ?= $(WABT_VERSION)
+
+wabt-setup: setup
+	$(call GIT_CLONE,https://github.com/WebAssembly/wabt.git,$(WABT_VERSION),wabt)
+	mkdir -p $(BUILD_WORK)/wabt/build
+
+ifneq ($(wildcard $(BUILD_WORK)/wabt/.build_complete),)
+wabt:
+	@echo "Using previously built wabt."
+else
+wabt: wabt-setup
+	cd $(BUILD_WORK)/wabt/build && cmake \
+		-DBUILD_TESTS=OFF $(DEFAULT_CMAKE_FLAGS) \
```

I think you forgot to do this...

```suggestion
		$(DEFAULT_CMAKE_FLAGS) \
		-DBUILD_TESTS=OFF \
```